src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -4,30 +4,20 @@
 // 1. If not already done, enable the Google Cloud Logging API
 //    and check the quota for your project at
 //    https://console.developers.google.com/apis/api/logging
-// 2. This sample uses Application Default Credentials for authentication.
+// 2. Install the Node.js client library by running
+//    `npm install googleapis --save`
```

Comment:
The ADC bit wasn't wrong, but given that the auth library is exported by the module, it didn't seem particularly useful to mention it